### PR TITLE
Updating to sync with SciDB 23.10

### PR DESCRIPTION
**Pull request by jlevine-p4**

### CODE REVIEW

**Comment by jlevine-p4** [comment]
Interesting. It failed a couple of tests because the workflow is still trying to use SciDB 22.6. The workflow files look like they point to version 23.2, I can update those to point to 23.10 and see if that helps.

File: docker/rocky-8/res/install-scidb.sh [hunk 1]
```diff
@@ -27,12 +27,12 @@ sed --in-place                                  \
     $P4_REPO
 
 sed --in-place                                  \
-    "s/22.5/22.6/"                              \
+    "s/22.6/23.10/"                              \
```
Comment (rvernica) [inline docker/rocky-8/res/install-scidb.sh:30]: This entire `sed` command should go away. Before this PR, I was downloading the 22.5 repo file (as there was not 22.6 repo) and replacing 22.5 with 22.6 in the repo file. 
Now you are downloading the 23.10 report and there is no need to replace anything.

File: install.sh [hunk 2]
```diff
@@ -7,8 +7,8 @@
 set -o errexit
 
 
-SCIDB_VER=22.6
-PKG_VER=8
+SCIDB_VER=23.10
+PKG_VER=0
```
Comment (rvernica) [inline install.sh:11]: We start with version 1

File: specs/extra-scidb-libs.spec [hunk 3]
```diff
@@ -1,5 +1,5 @@
-Name:           extra-scidb-libs-22.6
-Version:        8
+Name:           extra-scidb-libs-23.10
+Version:        0
```
Comment (rvernica) [inline specs/extra-scidb-libs.spec:2]: Version 1

**Review by rvernica — CHANGES_REQUESTED** [review]
Minor comments, please review

**Comment by rvernica** [comment]
Yes, update the workflow to use the 23.10 images. There are no 23.10 images yet. So, the workflow will still fail.
In parallel, I'll work on the 23.10 images and we should have this workflow succeed soon.

**Review by rvernica — APPROVED** [review]

**Comment by rvernica** [comment]
I merged the master and resolved conflicts. I made some additional fixes. The locally built package tests successfully.

**Pull request merged**